content/configuration/tasks-build.md

```diff
@@ -98,29 +98,4 @@ tasks:
 
 In order to build small images and/or hide secret values required during the build process, you can use artifacts. These artifacts will allow you to share files or folders between different build steps that use a combination of Dockerfiles, context and build arguments.
 
-![](/images/multi-step-building.png)
```

Comment:
You didn't like the image? 😜 Wouldn't be worth having an illustration for that, still?